install-dev/index_cli.php

```diff
@@ -40,6 +40,10 @@
     exit(0);
 } catch (PrestashopInstallerException $e) {
     $e->displayMessage();
+} catch (Throwable $t) {
+    // Executed only in PHP 7, will not match in PHP 5.
+    // Allows `Error` classes to be catched without throwing an error on PHP 5.
```

Comment:
You mean "on PHP 7+", don't you?

Reply:
Nupe, I wanted to say no error will be thrown on PHP 5.